queue_processors/queue_processor/validate_job.py

```diff
@@ -0,0 +1,60 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+"""
+Functions to ensure that the Reduction job is valid before attempting to process the data
+"""
+from nexusformat.nexus import nxload
+from nexusformat.nexus.tree import NeXusError
+
+
+def is_valid_rb(rb_number):
+    """
+    Detects if the RB number is valid e.g. (above 0 and not a string)
+    :param rb_number:
+    :return: An error message if one is generated or None if the RB is valid
+    """
+    try:
+        rb_number = int(rb_number)
+        if rb_number > 0:
+            return None
+        return "Calibration file detected (RB Number less than or equal to 0)"
+    except ValueError:
+        return "Calibration file detected (RB Number is a string)"
+
+
+def check_beam_current(run_file_location):
+    """
+    Ensure that that there is value for beam current in the file else we can assume the beam is off
+    :param run_file_location: The location of the run file to inspect
+    :return: An error message if one is generated or None if run has beam current
```

Comment:
Is it worth more accurately saying
"if run has beam current **greater than 0.1 (10uAmp)** "
or "if run has **significant** current"
?